src/docs/upgrade.dox

```diff
@@ -85,5 +85,8 @@ To downgrade to a previous release of WiredTiger:
 -# Shut down the new version of the application.
 <br><br>
 -# Start the old version of the application.
+<br><br>
+-# The database created with version 10.0 cannot be downgraded to older
+   versions.
```

Comment:
Can we be more verbose here? "Underlying file formats changed between the XXX release and the YYY release, which means that..."

Reply:
Thanks for flagging this. I have updated the document.